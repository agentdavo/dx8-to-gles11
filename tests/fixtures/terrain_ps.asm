ps.1.1

tex t0
tex t1
lrp r0, v0.a, t1, t0
mul r0, r0, v0
