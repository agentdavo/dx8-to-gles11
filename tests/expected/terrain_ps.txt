TEX_SAMPLE stage=0
TEX_SAMPLE stage=1
TEX_ENV_COMBINE func=34165
TEX_ENV_COMBINE func=8448
